bin/p2-shutdown/main.go

```diff
@@ -84,23 +86,22 @@ func main() {
 	haltWG.Wait()
 }
 
-func shutdownPod(podID types.PodID, podsToShutdown []types.PodID) bool {
-	if len(podsToShutdown) == 0 {
-		return true
+// returns true if the pod should be shutdown
+func shouldShutdownPod(podID types.PodID, podsToShutdown []types.PodID, podsToExcludeFromShutdown []types.PodID) bool {
+	for _, excludedPod := range podsToExcludeFromShutdown {
+		if excludedPod == podID {
+			return false
+		}
 	}
 	for _, pod := range podsToShutdown {
 		if pod == podID {
 			return true
 		}
 	}
-	return false
-}
 
-func listInclude(pod string, pods []string) bool {
-	for _, p := range pods {
-		if p == pod {
-			return true
-		}
+	if len(podsToShutdown) > 0 {
```

Comment:
I guess if you wanted to code golf you could `return len(podsToShutdown) == 0`

shrug